### PR TITLE
Add scikits_odes_sundials

### PR DESCRIPTION
**Pull request by c-randall**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by c-randall** [comment]
I am ok being listed as the maintainer.

**Comment by c-randall** [comment]
@conda-forge-admin, please ping team
@conda-forge/help-python, ready for review

**Comment by c-randall** [comment]
Re-ping now that it has been 2 weeks.

@conda-forge-admin, please ping team
@conda-forge/help-python, ready for review

File: recipes/scikits_odes_sundials/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 277d0400812f63a11624565d82f9ecc7497263ebfb9f049e5ca8ab2d025c5621
+
+build:
+  script: {{ PYTHON }} -m pip install .
```
Comment (ocefpaf) [inline recipes/scikits_odes_sundials/meta.yaml:13]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

**Pull request merged**